src/DependencyInjection/Configuration.php

```diff
@@ -141,6 +141,10 @@ public function getConfigTreeBuilder()
+                            ->info('Custom default mosaic background')
```

Comment:
`Background used in mosaic view (base64 encoded)`

Reply:
It also can be an url, so I just used `Background used in mosaic view`